Script started on 2019-05-17 16:14:52+0200
(base) ]0;ironbas3@ironbas3-Hyrican-PC: ~/utils[01;32mironbas3@ironbas3-Hyrican-PC[00m:[01;34m~/utils[00m$ conda activate deeep[K[KpspeechTrain
(deepspeechTrain) ]0;ironbas3@ironbas3-Hyrican-PC: ~/utils[01;32mironbas3@ironbas3-Hyrican-PC[00m:[01;34m~/utils[00m$ conda activate deepspeechTrainexit[K[1Pdirpython -u generate_deepspeech_train.py && exit
>>>>>>>>>>>>>>>>.. INCREASING NUMBER OF EPOCHS DEFAULT >>>>>>>>>>>>>>>>>>>>
>>>>>>>>>>>>>>>>.. INCREASING NUMBER OF EPOCHS DEFAULT >>>>>>>>>>>>>>>>>>>>
>>>>>>>>>>>>>>>>.. INCREASING NUMBER OF EPOCHS DEFAULT >>>>>>>>>>>>>>>>>>>>
Training number 1
Version directory already exists
>>> Training with version_num: n_hidden_1024_dp_0.4
b"Preprocessing ['/mnt/datasets/de/clips/train.csv']"
b'Preprocessing done'
b"Preprocessing ['/mnt/datasets/de/clips/dev.csv']"
b'Preprocessing done'
b'I STARTING Optimization'
b'I Training epoch 30...'
N/A% (0 of 415) |                        | Elapsed Time: 0:00:00 ETA:  --:--:--
100% (415 of 415) |######################| Elapsed Time: 0:01:04 ETA:  00:00:00
b'I Training of Epoch 30 - loss: 48.827615'
100% (415 of 415) |######################| Elapsed Time: 0:01:04 Time:  0:01:04
b'I Validating epoch 30...'
N/A% (0 of 180) |                        | Elapsed Time: 0:00:00 ETA:  --:--:--
100% (180 of 180) |######################| Elapsed Time: 0:00:11 ETA:  00:00:00
b'I Validation of Epoch 30 - loss: 90.655996'
100% (180 of 180) |######################| Elapsed Time: 0:00:11 Time:  0:00:11
b'I Training epoch 31...'
N/A% (0 of 415) |                        | Elapsed Time: 0:00:00 ETA:  --:--:--
100% (415 of 415) |######################| Elapsed Time: 0:01:06 ETA:  00:00:00
b'I Training of Epoch 31 - loss: 49.785081'
100% (415 of 415) |######################| Elapsed Time: 0:01:06 Time:  0:01:06
b'I Validating epoch 31...'
N/A% (0 of 180) |                        | Elapsed Time: 0:00:00 ETA:  --:--:--
100% (180 of 180) |######################| Elapsed Time: 0:00:12 ETA:  00:00:00
b'I Validation of Epoch 31 - loss: 97.257398'
100% (180 of 180) |######################| Elapsed Time: 0:00:12 Time:  0:00:12
b'I Training epoch 32...'
N/A% (0 of 415) |                        | Elapsed Time: 0:00:00 ETA:  --:--:--
100% (415 of 415) |######################| Elapsed Time: 0:01:06 ETA:  00:00:00
b'I Training of Epoch 32 - loss: 48.672885'
100% (415 of 415) |######################| Elapsed Time: 0:01:06 Time:  0:01:06
b'I Validating epoch 32...'
N/A% (0 of 180) |                        | Elapsed Time: 0:00:00 ETA:  --:--:--
100% (180 of 180) |######################| Elapsed Time: 0:00:11 ETA:  00:00:00
b'I Validation of Epoch 32 - loss: 98.014308'
100% (180 of 180) |######################| Elapsed Time: 0:00:12 Time:  0:00:12
b'I Training epoch 33...'
N/A% (0 of 415) |                        | Elapsed Time: 0:00:00 ETA:  --:--:--
100% (415 of 415) |######################| Elapsed Time: 0:01:06 ETA:  00:00:00
b'I Training of Epoch 33 - loss: 46.967278'
100% (415 of 415) |######################| Elapsed Time: 0:01:06 Time:  0:01:06
b'I Validating epoch 33...'
N/A% (0 of 180) |                        | Elapsed Time: 0:00:00 ETA:  --:--:--
100% (180 of 180) |######################| Elapsed Time: 0:00:12 ETA:  00:00:00
b'I Validation of Epoch 33 - loss: 100.398975'
b'I Early stop triggered as (for last 4 steps) validation loss: 100.398975 with standard deviation: 3.304814 and mean: 95.309234'
b'I FINISHED Optimization - training time: 0:05:12'
100% (180 of 180) |######################| Elapsed Time: 0:00:12 Time:  0:00:12
b"Preprocessing ['/mnt/datasets/de/clips/test.csv']"
b'Preprocessing done'
b'Computing acoustic model predictions...'
N/A% (0 of 178) |                        | Elapsed Time: 0:00:00 ETA:  --:--:--
100% (178 of 178) |######################| Elapsed Time: 0:00:20 Time:  0:00:20
b'Decoding predictions...'
N/A% (0 of 178) |                        | Elapsed Time: 0:00:00 ETA:  --:--:--
100% (178 of 178) |######################| Elapsed Time: 0:01:18 Time:  0:01:18
b'Test - WER: 0.848945, CER: 26.119382, loss: 107.865547'
b'--------------------------------------------------------------------------------'
b'WER: 2.000000, CER: 8.000000, loss: 25.167969'
b' - src: "lesezeichen anzeigen"'
b' - res: "wieder teichen an einen"'
b'--------------------------------------------------------------------------------'
b'WER: 2.000000, CER: 16.000000, loss: 52.418491'
b' - src: "zur bibliothek hinzufugen"'
b' - res: "so wie sie sie in zugegen"'
b'--------------------------------------------------------------------------------'
b'WER: 1.750000, CER: 44.000000, loss: 180.085785'
b' - src: "provisorisch aufgestellte wohncontainer dienen den neuankommlingen als unterkunft"'
b' - res: "ob es euch auf gescheiter kommt in einen den neun a in den und oft"'
b'--------------------------------------------------------------------------------'
b'WER: 1.666667, CER: 13.000000, loss: 59.509411'
b' - src: "gehort das so"'
b' - res: "der was du an seine"'
b'--------------------------------------------------------------------------------'
b'WER: 1.600000, CER: 22.000000, loss: 68.466690'
b' - src: "im erzgebirge wurden lithiumvorkommen entdeckt"'
b' - res: "die mergelige wurde ihm vor am man eine"'
b'--------------------------------------------------------------------------------'
b'WER: 1.600000, CER: 26.000000, loss: 140.006454'
b' - src: "vermutlich ist die zylinderkopfdichtung undicht"'
b' - res: "deutlich die zu den de auf die schon endlich"'
b'--------------------------------------------------------------------------------'
b'WER: 1.500000, CER: 4.000000, loss: 11.094448'
b' - src: "weitermachen bitte"'
b' - res: "weiter machen wie"'
b'--------------------------------------------------------------------------------'
b'WER: 1.500000, CER: 6.000000, loss: 14.894795'
b' - src: "alles entfernen"'
b' - res: "als ein fern"'
b'--------------------------------------------------------------------------------'
b'WER: 1.500000, CER: 11.000000, loss: 19.350058'
b' - src: "wunschliste anzeigen"'
b' - res: "mischet an sein"'
b'--------------------------------------------------------------------------------'
b'WER: 1.500000, CER: 6.000000, loss: 23.507359'
b' - src: "einstellungen anzeigen"'
b' - res: "einstellen an ein"'
b'--------------------------------------------------------------------------------'
b'I Exporting the model...'
b'I Models exported at /home/ironbas3/Past_Models/de_n_hidden_1024_dp_0.4/de_model_n_hidden_1024_dp_0.4'
>>>>>>>>>>>>>>>>.. DECREASING BATCH SIZE >>>>>>>>>>>>>>>>>>>>
>>>>>>>>>>>>>>>>.. DECREASING BATCH SIZE >>>>>>>>>>>>>>>>>>>>
>>>>>>>>>>>>>>>>.. DECREASING BATCH SIZE >>>>>>>>>>>>>>>>>>>>
Training number 2
Version directory already exists
>>> Training with version_num: n_hidden_1024_dp_0.2
Traceback (most recent call last):
  File "/home/ironbas3/anaconda3/envs/deepspeechTrain/lib/python3.6/site-packages/absl/flags/_flag.py", line 181, in _parse
    return self.parser.parse(argument)
  File "/home/ironbas3/anaconda3/envs/deepspeechTrain/lib/python3.6/site-packages/absl/flags/_argument_parser.py", line 152, in parse
    val = self.convert(argument)
  File "/home/ironbas3/anaconda3/envs/deepspeechTrain/lib/python3.6/site-packages/absl/flags/_argument_parser.py", line 265, in convert
    return int(argument, base)
ValueError: invalid literal for int() with base 10: '3.0'
During handling of the above exception, another exception occurred:
Traceback (most recent call last):
  File "/home/ironbas3/DeepSpeech/DeepSpeech.py", line 941, in <module>
    tf.app.run(main)
  File "/home/ironbas3/anaconda3/envs/deepspeechTrain/lib/python3.6/site-packages/tensorflow/python/platform/app.py", line 119, in run
    argv = flags.FLAGS(_sys.argv if argv is None else argv, known_only=True)
  File "/home/ironbas3/anaconda3/envs/deepspeechTrain/lib/python3.6/site-packages/tensorflow/python/platform/flags.py", line 112, in __call__
    return self.__dict__['__wrapped'].__call__(*args, **kwargs)
  File "/home/ironbas3/anaconda3/envs/deepspeechTrain/lib/python3.6/site-packages/absl/flags/_flagvalues.py", line 626, in __call__
    unknown_flags, unparsed_args = self._parse_args(args, known_only)
  File "/home/ironbas3/anaconda3/envs/deepspeechTrain/lib/python3.6/site-packages/absl/flags/_flagvalues.py", line 774, in _parse_args
    flag.parse(value)
  File "/home/ironbas3/anaconda3/envs/deepspeechTrain/lib/python3.6/site-packages/absl/flags/_flag.py", line 166, in parse
    self.value = self._parse(argument)
  File "/home/ironbas3/anaconda3/envs/deepspeechTrain/lib/python3.6/site-packages/absl/flags/_flag.py", line 184, in _parse
    'flag --%s=%s: %s' % (self.name, argument, e))
absl.flags._exceptions.IllegalFlagValueError: flag --train_batch_size=3.0: invalid literal for int() with base 10: '3.0'
Training number 3
Version directory already exists
>>> Training with version_num: n_hidden_2048_dp_0.2
Traceback (most recent call last):
  File "/home/ironbas3/anaconda3/envs/deepspeechTrain/lib/python3.6/site-packages/absl/flags/_flag.py", line 181, in _parse
    return self.parser.parse(argument)
  File "/home/ironbas3/anaconda3/envs/deepspeechTrain/lib/python3.6/site-packages/absl/flags/_argument_parser.py", line 152, in parse
    val = self.convert(argument)
  File "/home/ironbas3/anaconda3/envs/deepspeechTrain/lib/python3.6/site-packages/absl/flags/_argument_parser.py", line 265, in convert
    return int(argument, base)
ValueError: invalid literal for int() with base 10: '3.0'
During handling of the above exception, another exception occurred:
Traceback (most recent call last):
  File "/home/ironbas3/DeepSpeech/DeepSpeech.py", line 941, in <module>
    tf.app.run(main)
  File "/home/ironbas3/anaconda3/envs/deepspeechTrain/lib/python3.6/site-packages/tensorflow/python/platform/app.py", line 119, in run
    argv = flags.FLAGS(_sys.argv if argv is None else argv, known_only=True)
  File "/home/ironbas3/anaconda3/envs/deepspeechTrain/lib/python3.6/site-packages/tensorflow/python/platform/flags.py", line 112, in __call__
    return self.__dict__['__wrapped'].__call__(*args, **kwargs)
  File "/home/ironbas3/anaconda3/envs/deepspeechTrain/lib/python3.6/site-packages/absl/flags/_flagvalues.py", line 626, in __call__
    unknown_flags, unparsed_args = self._parse_args(args, known_only)
  File "/home/ironbas3/anaconda3/envs/deepspeechTrain/lib/python3.6/site-packages/absl/flags/_flagvalues.py", line 774, in _parse_args
    flag.parse(value)
  File "/home/ironbas3/anaconda3/envs/deepspeechTrain/lib/python3.6/site-packages/absl/flags/_flag.py", line 166, in parse
    self.value = self._parse(argument)
  File "/home/ironbas3/anaconda3/envs/deepspeechTrain/lib/python3.6/site-packages/absl/flags/_flag.py", line 184, in _parse
    'flag --%s=%s: %s' % (self.name, argument, e))
absl.flags._exceptions.IllegalFlagValueError: flag --train_batch_size=3.0: invalid literal for int() with base 10: '3.0'
Training number 4
Version directory already exists
>>> Training with version_num: n_hidden_1024_dp_0.3
Traceback (most recent call last):
  File "/home/ironbas3/anaconda3/envs/deepspeechTrain/lib/python3.6/site-packages/absl/flags/_flag.py", line 181, in _parse
    return self.parser.parse(argument)
  File "/home/ironbas3/anaconda3/envs/deepspeechTrain/lib/python3.6/site-packages/absl/flags/_argument_parser.py", line 152, in parse
    val = self.convert(argument)
  File "/home/ironbas3/anaconda3/envs/deepspeechTrain/lib/python3.6/site-packages/absl/flags/_argument_parser.py", line 265, in convert
    return int(argument, base)
ValueError: invalid literal for int() with base 10: '3.0'
During handling of the above exception, another exception occurred:
Traceback (most recent call last):
  File "/home/ironbas3/DeepSpeech/DeepSpeech.py", line 941, in <module>
    tf.app.run(main)
  File "/home/ironbas3/anaconda3/envs/deepspeechTrain/lib/python3.6/site-packages/tensorflow/python/platform/app.py", line 119, in run
    argv = flags.FLAGS(_sys.argv if argv is None else argv, known_only=True)
  File "/home/ironbas3/anaconda3/envs/deepspeechTrain/lib/python3.6/site-packages/tensorflow/python/platform/flags.py", line 112, in __call__
    return self.__dict__['__wrapped'].__call__(*args, **kwargs)
  File "/home/ironbas3/anaconda3/envs/deepspeechTrain/lib/python3.6/site-packages/absl/flags/_flagvalues.py", line 626, in __call__
    unknown_flags, unparsed_args = self._parse_args(args, known_only)
  File "/home/ironbas3/anaconda3/envs/deepspeechTrain/lib/python3.6/site-packages/absl/flags/_flagvalues.py", line 774, in _parse_args
    flag.parse(value)
  File "/home/ironbas3/anaconda3/envs/deepspeechTrain/lib/python3.6/site-packages/absl/flags/_flag.py", line 166, in parse
    self.value = self._parse(argument)
  File "/home/ironbas3/anaconda3/envs/deepspeechTrain/lib/python3.6/site-packages/absl/flags/_flag.py", line 184, in _parse
    'flag --%s=%s: %s' % (self.name, argument, e))
absl.flags._exceptions.IllegalFlagValueError: flag --train_batch_size=3.0: invalid literal for int() with base 10: '3.0'
Training number 5
Version directory already exists
>>> Training with version_num: n_hidden_2048_dp_0.3
Traceback (most recent call last):
  File "/home/ironbas3/anaconda3/envs/deepspeechTrain/lib/python3.6/site-packages/absl/flags/_flag.py", line 181, in _parse
    return self.parser.parse(argument)
  File "/home/ironbas3/anaconda3/envs/deepspeechTrain/lib/python3.6/site-packages/absl/flags/_argument_parser.py", line 152, in parse
    val = self.convert(argument)
  File "/home/ironbas3/anaconda3/envs/deepspeechTrain/lib/python3.6/site-packages/absl/flags/_argument_parser.py", line 265, in convert
    return int(argument, base)
ValueError: invalid literal for int() with base 10: '3.0'
During handling of the above exception, another exception occurred:
Traceback (most recent call last):
  File "/home/ironbas3/DeepSpeech/DeepSpeech.py", line 941, in <module>
    tf.app.run(main)
  File "/home/ironbas3/anaconda3/envs/deepspeechTrain/lib/python3.6/site-packages/tensorflow/python/platform/app.py", line 119, in run
    argv = flags.FLAGS(_sys.argv if argv is None else argv, known_only=True)
  File "/home/ironbas3/anaconda3/envs/deepspeechTrain/lib/python3.6/site-packages/tensorflow/python/platform/flags.py", line 112, in __call__
    return self.__dict__['__wrapped'].__call__(*args, **kwargs)
  File "/home/ironbas3/anaconda3/envs/deepspeechTrain/lib/python3.6/site-packages/absl/flags/_flagvalues.py", line 626, in __call__
    unknown_flags, unparsed_args = self._parse_args(args, known_only)
  File "/home/ironbas3/anaconda3/envs/deepspeechTrain/lib/python3.6/site-packages/absl/flags/_flagvalues.py", line 774, in _parse_args
    flag.parse(value)
  File "/home/ironbas3/anaconda3/envs/deepspeechTrain/lib/python3.6/site-packages/absl/flags/_flag.py", line 166, in parse
    self.value = self._parse(argument)
  File "/home/ironbas3/anaconda3/envs/deepspeechTrain/lib/python3.6/site-packages/absl/flags/_flag.py", line 184, in _parse
    'flag --%s=%s: %s' % (self.name, argument, e))
absl.flags._exceptions.IllegalFlagValueError: flag --train_batch_size=3.0: invalid literal for int() with base 10: '3.0'
Training number 6
Version directory already exists
>>> Training with version_num: n_hidden_2048_dp_0.4
Traceback (most recent call last):
  File "/home/ironbas3/anaconda3/envs/deepspeechTrain/lib/python3.6/site-packages/absl/flags/_flag.py", line 181, in _parse
    return self.parser.parse(argument)
  File "/home/ironbas3/anaconda3/envs/deepspeechTrain/lib/python3.6/site-packages/absl/flags/_argument_parser.py", line 152, in parse
    val = self.convert(argument)
  File "/home/ironbas3/anaconda3/envs/deepspeechTrain/lib/python3.6/site-packages/absl/flags/_argument_parser.py", line 265, in convert
    return int(argument, base)
ValueError: invalid literal for int() with base 10: '3.0'
During handling of the above exception, another exception occurred:
Traceback (most recent call last):
  File "/home/ironbas3/DeepSpeech/DeepSpeech.py", line 941, in <module>
    tf.app.run(main)
  File "/home/ironbas3/anaconda3/envs/deepspeechTrain/lib/python3.6/site-packages/tensorflow/python/platform/app.py", line 119, in run
    argv = flags.FLAGS(_sys.argv if argv is None else argv, known_only=True)
  File "/home/ironbas3/anaconda3/envs/deepspeechTrain/lib/python3.6/site-packages/tensorflow/python/platform/flags.py", line 112, in __call__
    return self.__dict__['__wrapped'].__call__(*args, **kwargs)
  File "/home/ironbas3/anaconda3/envs/deepspeechTrain/lib/python3.6/site-packages/absl/flags/_flagvalues.py", line 626, in __call__
    unknown_flags, unparsed_args = self._parse_args(args, known_only)
  File "/home/ironbas3/anaconda3/envs/deepspeechTrain/lib/python3.6/site-packages/absl/flags/_flagvalues.py", line 774, in _parse_args
    flag.parse(value)
  File "/home/ironbas3/anaconda3/envs/deepspeechTrain/lib/python3.6/site-packages/absl/flags/_flag.py", line 166, in parse
    self.value = self._parse(argument)
  File "/home/ironbas3/anaconda3/envs/deepspeechTrain/lib/python3.6/site-packages/absl/flags/_flag.py", line 184, in _parse
    'flag --%s=%s: %s' % (self.name, argument, e))
absl.flags._exceptions.IllegalFlagValueError: flag --train_batch_size=3.0: invalid literal for int() with base 10: '3.0'
Training number 7
Version directory already exists
>>> Training with version_num: n_hidden_4096_dp_0.4
Traceback (most recent call last):
  File "/home/ironbas3/anaconda3/envs/deepspeechTrain/lib/python3.6/site-packages/absl/flags/_flag.py", line 181, in _parse
    return self.parser.parse(argument)
  File "/home/ironbas3/anaconda3/envs/deepspeechTrain/lib/python3.6/site-packages/absl/flags/_argument_parser.py", line 152, in parse
    val = self.convert(argument)
  File "/home/ironbas3/anaconda3/envs/deepspeechTrain/lib/python3.6/site-packages/absl/flags/_argument_parser.py", line 265, in convert
    return int(argument, base)
ValueError: invalid literal for int() with base 10: '3.0'
During handling of the above exception, another exception occurred:
Traceback (most recent call last):
  File "/home/ironbas3/DeepSpeech/DeepSpeech.py", line 941, in <module>
    tf.app.run(main)
  File "/home/ironbas3/anaconda3/envs/deepspeechTrain/lib/python3.6/site-packages/tensorflow/python/platform/app.py", line 119, in run
    argv = flags.FLAGS(_sys.argv if argv is None else argv, known_only=True)
  File "/home/ironbas3/anaconda3/envs/deepspeechTrain/lib/python3.6/site-packages/tensorflow/python/platform/flags.py", line 112, in __call__
    return self.__dict__['__wrapped'].__call__(*args, **kwargs)
  File "/home/ironbas3/anaconda3/envs/deepspeechTrain/lib/python3.6/site-packages/absl/flags/_flagvalues.py", line 626, in __call__
    unknown_flags, unparsed_args = self._parse_args(args, known_only)
  File "/home/ironbas3/anaconda3/envs/deepspeechTrain/lib/python3.6/site-packages/absl/flags/_flagvalues.py", line 774, in _parse_args
    flag.parse(value)
  File "/home/ironbas3/anaconda3/envs/deepspeechTrain/lib/python3.6/site-packages/absl/flags/_flag.py", line 166, in parse
    self.value = self._parse(argument)
  File "/home/ironbas3/anaconda3/envs/deepspeechTrain/lib/python3.6/site-packages/absl/flags/_flag.py", line 184, in _parse
    'flag --%s=%s: %s' % (self.name, argument, e))
absl.flags._exceptions.IllegalFlagValueError: flag --train_batch_size=3.0: invalid literal for int() with base 10: '3.0'
Training number 8
Version directory already exists
>>> Training with version_num: n_hidden_2048_dp_0.5
b"Preprocessing ['/mnt/datasets/de/clips/train.csv']"
b'Preprocessing done'
b"Preprocessing ['/mnt/datasets/de/clips/dev.csv']"
b'Preprocessing done'
b'I STARTING Optimization'
b'I Training epoch 23...'
N/A% (0 of 415) |                        | Elapsed Time: 0:00:00 ETA:  --:--:--
100% (415 of 415) |######################| Elapsed Time: 0:04:02 ETA:  00:00:00
b'I Training of Epoch 23 - loss: 45.330427'
100% (415 of 415) |######################| Elapsed Time: 0:04:02 Time:  0:04:02
b'I Validating epoch 23...'
N/A% (0 of 180) |                        | Elapsed Time: 0:00:00 ETA:  --:--:--
100% (180 of 180) |######################| Elapsed Time: 0:00:29 ETA:  00:00:00
b'I Validation of Epoch 23 - loss: 107.116271'
100% (180 of 180) |######################| Elapsed Time: 0:00:30 Time:  0:00:30
b'I Training epoch 24...'
N/A% (0 of 415) |                        | Elapsed Time: 0:00:00 ETA:  --:--:--
100% (415 of 415) |######################| Elapsed Time: 0:04:12 ETA:  00:00:00
b'I Training of Epoch 24 - loss: 45.772376'
100% (415 of 415) |######################| Elapsed Time: 0:04:12 Time:  0:04:12
b'I Validating epoch 24...'
N/A% (0 of 180) |                        | Elapsed Time: 0:00:00 ETA:  --:--:--
100% (180 of 180) |######################| Elapsed Time: 0:00:31 ETA:  00:00:00
b'I Validation of Epoch 24 - loss: 111.424512'
100% (180 of 180) |######################| Elapsed Time: 0:00:32 Time:  0:00:32
b'I Training epoch 25...'
N/A% (0 of 415) |                        | Elapsed Time: 0:00:00 ETA:  --:--:--
